app/middlewares/validators/userAlbumsValidator.js

```diff
@@ -0,0 +1,16 @@
+const { User } = require('../../models');
+const errors = require('../../errors');
+const userService = require('../../services/users');
+
+exports.validate = async (req, res, next) => {
+  try {
+    const user = await User.findByPk(req.params.id);
+    const tokenUser = await userService.findUserByMail(req.userToken.mail);
+    if (!(user.id === tokenUser.id || tokenUser.type === 'admin')) {
```

Comment:
Does this apply to each request? This seems a generic middleware but has a specific case use (check admin permission and need a path param)

Reply:
For now it only applies to this one, but I could give it a more generic name